src/index.tsx

```diff
@@ -276,20 +297,20 @@ function generateDraw(element: ExcaliburElement) {
       generator.line(x4, y4, x2, y2)
     ];
 
-    element.draw = (rc, context) => {
-      context.translate(element.x, element.y);
+    element.draw = function(rc, context) {
+      context.translate(this.x, this.y);
       shapes.forEach(shape => rc.draw(shape));
-      context.translate(-element.x, -element.y);
+      context.translate(-this.x, -this.y);
     };
     return;
   } else if (isTextElement(element)) {
-    element.draw = (rc, context) => {
+    element.draw = function(rc, context) {
       const font = context.font;
-      context.font = element.font;
+      context.font = this.font;
       context.fillText(
-        element.text,
-        element.x,
-        element.y + element.actualBoundingBoxAscent
+        this.text,
```

Comment:
Because we copy over the element, we can't capture it and have to use the one passed when draw is called.